redis/reply.go

```diff
@@ -477,3 +478,55 @@ func Positions(result interface{}, err error) ([]*[2]float64, error) {
 	}
 	return positions, nil
 }
+
+// SlowLogs is a helper that parse the SLOWLOG GET command output and
+// return the array of SlowLog
+func SlowLogs(result interface{}, err error) ([]*SlowLog, error) {
```

Comment:
Any reason this is a `*SlowLog` instead of just a `SlowLog`?

Reply:
As per discussion with Gary Burd we agreed upon this. Please refer to : https://github.com/gomodule/redigo/pull/414#issuecomment-476484973

Reply:
I think you've taken Gary's suggestion on the improvements too literally, I can't see any point that he said "return as `[]*SlowLog`", he just gave you an example of how to approach.

As an informational type much the same way `time.Time` is used we should avoid the overhead of allocating a pointer which will never be used and [can cause issues in the usage](https://medium.com/codezillas/uh-ohs-in-go-slice-of-pointers-c0a30669feee).

The fix is simple I've made the suggestions for this.